src/Core/Label.js

```diff
@@ -49,12 +49,14 @@ class Label extends THREE.Object3D {
      * is applied, it cannot be changed directly. However, if it really needed,
      * it can be accessed through `label.content.style`, but it is highly
      * discouraged to do so.
-     * @param {Object} [sprites] the sprites.
      */
-    constructor(content = '', coordinates, style = {}, sprites) {
+    constructor(content = '', coordinates, style = {}, ...args) {
         if (coordinates == undefined) {
             throw new Error('coordinates are mandatory to add a Label');
         }
+        if (args.length > 0) {
```

Comment:
I suggest the following instead to avoid polluting the constructor signature:
```
if (arguments.length > 3) {
    console.log('Deprecated argument sprites in Label constructor. Sprites must be configured in style argument.');
}
```

Reply:
done. I was using the same methode as i was doing elsewhere. But here we can use arguments as we don't use any of the argument objects.
Thanks for the proposal